crates/rust-analyzer/src/global_state.rs

```diff
@@ -34,6 +36,7 @@ use crate::{
 pub(crate) enum Status {
     Loading,
     Ready,
+    ReadyPartial,
```

Comment:
For the lsp_ext part, I think two separate enum variants makes sense. For this status however, I thing `Ready { partial: bool }` would work better?

Reply:
Fixed